src/python/zquantum/core/interfaces/estimator.py

```diff
@@ -27,8 +31,33 @@ def get_estimated_expectation_values(
             backend (QuantumBackend): The backend used to run quantum circuits, either a simulator or quantum hardware.
             circuit (Circuit): The circuit that prepares the unknown quantum state.
             target_operator (List[SymbolicOperator]): List of target functions to be estimated.
+            n_samples (int): Number of measurements done on the unknown quantum state. 
+            epsilon (float): an error term.
+            delta (float): a confidence term.
 
         Returns:
             ExpectationValues: The estimations of the terms in the target_operator. 
         """
         raise NotImplementedError
+
+    def _ignore_parameter(
```

Comment:
Don't you think something like "_log_ignore_parameter" or "_log_parameter_ignored" is a better name?